wiki/People/Beatmap_Nominators/Beatmap_Veto/en.md

```diff
@@ -0,0 +1,56 @@
+_**Main page:  [Beatmap Nominator Rules](https://osu.ppy.sh/help/wiki/People/Beatmap_Nominators/Rules)**_
+
+The _beatmap veto_ allows a [Beatmap Nominator](https://osu.ppy.sh/help/wiki/People/Beatmap_Nominators) to withhold a beatmap from Qualification if they feel there are significant issues regarding beatmap quality which make it unfit for the Ranked section.
+
+Seeking further discussion or clarification over any kind of quality issues which you feel need to be addressed before the map can proceed with Qualification is one of your main responsibilities. 
+
+Any Beatmap Nominator may place a beatmap veto, provided they are formally qualified for the game mode(s) in question, as stated in the [Beatmap Nominator Rules](https://osu.ppy.sh/help/wiki/People/Beatmap_Nominators/Rules).
+
+Each beatmap veto must be discussed by both parties - the mapper as well as the Nominator placing the beatmap veto - in an attempt to reach an agreed compromise.
+
+In cases where discussion does not take place or is inconclusive, members of the [Quality Assurance Team](https://osu.ppy.sh/groups/7)’s disqualification branch will mediate to either uphold or dismiss the veto.
```

Comment:
```diff
- [Quality Assurance Team](https://osu.ppy.sh/groups/7)
+ [QAT](/wiki/Quality_Assurance_Team) (Quality Assurance Team)
```
This is what I'm suggesting for that part:

- Change this to QAT followed by Quality Assurance Team in round braces, from this point forward, only use QAT.
- Change the link to the wiki page.

Reply:
yeah this saves wordiness later on since I don't have to print the whole title each time